app/src/features/projects/components/ProjectFundingItemForm.tsx

```diff
@@ -71,17 +126,34 @@ export interface IProjectFundingItemFormProps {
  * @param {*} props
  * @return {*}
  */
-
 const ProjectFundingItemForm: React.FC<IProjectFundingItemFormProps> = (props) => {
   const formikProps = useFormikContext<IProjectFundingFormArrayItem>();
-
   const { values, touched, errors, handleChange, handleSubmit, setFieldValue } = formikProps;
-
   // Only show investment_action_category if certain agency_id values are selected
   // Toggle investment_action_category label and dropdown values based on chosen agency_id
   const investment_action_category_label =
     (values.agency_id === 1 && 'Investment Action') || (values.agency_id === 2 && 'Investment Category') || null;
 
+  const findItemLabel = (id: number, type: FundingSourceType) => {
+    return props.sources.find((item) => item.value === id && item.type === type)?.label;
+  };
+
+  // find label for initial value
+  const mapInitialValue = (
+    formValues?: IProjectFundingFormArrayItem
+  ): IAutocompleteFieldOptionWithType<number> | undefined => {
+    if (formValues) {
+      const initialValue = {
+        value: formValues.agency_id ?? formValues.first_nations_id,
+        label: '',
+        type: formValues.agency_id ? FundingSourceType.FUNDING_SOURCE : FundingSourceType.FIRST_NATIONS
+      } as IAutocompleteFieldOptionWithType<number>;
+
+      initialValue.label = `${findItemLabel(initialValue.value, initialValue.type)}`;
```

Comment:
Could we wrap this in a `String` constructor instead of using a template string (if the goal is to coerce the return value to be a string)?

Reply:
If you're not using it to build strings comprising a programatic part and a literal part (e.g. something like `Hello, ${name}), it's preferrable to avoid it because
  - it adds unneccary computation
  - the syntax makes it easy to add whitespace or other accidental characters
  - in the case where you're just using it to coerce a value into a string, it's not immediately obvious what you're doing. The `String` constructor makes it very clear that you're converting the object to a string (and constructors in Javascript often have better browser support)

Reply:
I've updated that object creation to use a String()